src/SPC/builder/Extension.php

```diff
@@ -384,20 +401,18 @@ public function buildShared(): void
      */
     public function buildUnixShared(): void
     {
-        $config = (new SPCConfigUtil($this->builder))->config([$this->getName()], with_dependencies: true);
-        [$staticLibString, $sharedLibString] = $this->getStaticAndSharedLibs();
-
-        // macOS ld64 doesn't understand these, while Linux and BSD do
-        // use them to make sure that all symbols are picked up, even if a library has already been visited before
-        $preStatic = PHP_OS_FAMILY !== 'Darwin' ? '-Wl,-Bstatic -Wl,--start-group ' : '';
-        $postStatic = PHP_OS_FAMILY !== 'Darwin' ? ' -Wl,--end-group -Wl,-Bdynamic ' : ' ';
+        $config = (new SPCConfigUtil($this->builder))->config([$this->getName()]);
```

Comment:
looks like it's missing `'libs_only_deps' => true` ?

Reply:
No, extensions need to link against libphp.

Reply:
Oh, it must be outside the group though. Reworked the static/shared string logic and forgot about libphp.a existing.